### PR TITLE
data-migration: add dmctl command mode

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

Add new feature (command mode) introduction about dm.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v4.0 changes for now)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/1901
- Other reference link(s):

### CODE REVIEW

File: reference/tools/data-migration/manage-tasks.md [hunk 1]
```diff
@@ -8,9 +8,11 @@ category: reference
 
 This document describes how to manage and maintain the data replication task using the [dmctl](/reference/tools/data-migration/overview.md#dmctl) component. For the Data Migration cluster deployed using DM-Ansible, the dmctl binary file is in `dm-ansible/dmctl`.
 
-## dmctl basic usage
+The dmctl support the interactive mode for manual operation, and support the command mode for the script.
```
Comment (TomShawn) [inline reference/tools/data-migration/manage-tasks.md]:
```suggestion
The dmctl component supports the interactive mode for manual operations, and also supports the command mode for the script.
```

File: reference/tools/data-migration/manage-tasks.md [hunk 2]
```diff
@@ -8,9 +8,11 @@ category: reference
...
 
-This section shows the basic usage of dmctl commands.
+## dmctl interactive mode
+
+This section described the basic use of dmctl commands in interactive mode.
```
Comment (TomShawn) [inline reference/tools/data-migration/manage-tasks.md]:
```suggestion
This section described the basic use of dmctl commands in the interactive mode.
```

File: reference/tools/data-migration/manage-tasks.md [hunk 3]
```diff
@@ -817,6 +825,49 @@ You can use the `refresh-worker-tasks` command to forcefully refresh the `task =
 >
 > Normally, you do not need to use this command. Use it only when you are sure that the `task => DM-workers` mapping exists, but you are still prompted to refresh while you are executing other commands.
 
+## dmctl command mode
+
+The command mode differs from the interactive mode in that you need to perform the task operation right after executing the dmctl command. The parameters of the task operation in command mode is the same as that in interactive mode.
```
Comment (TomShawn) [inline reference/tools/data-migration/manage-tasks.md]: @toutdesuite Please check with the author whether `命令模式跟交互模式的区别是，执行命令时只需要在 dmctl 命令后紧接着执行任务操作，任务操作同交互模式的参数一致。` means the edited version, thanks!
```suggestion
The command mode differs from the interactive mode in that you need to append the task operation right after the dmctl command. The parameters of the task operation in the command mode are the same as those in the interactive mode.
```

File: reference/tools/data-migration/manage-tasks.md [hunk 4]
```diff
@@ -817,6 +825,49 @@ You can use the `refresh-worker-tasks` command to forcefully refresh the `task =
...
+
+> **Note:**
+>
+> + A dmctl command follows a task command only.
```
Comment (TomShawn) [inline reference/tools/data-migration/manage-tasks.md]:
```suggestion
> + A dmctl command must be followed by only one task operation.
```

File: reference/tools/data-migration/manage-tasks.md [hunk 5]
```diff
@@ -817,6 +825,49 @@ You can use the `refresh-worker-tasks` command to forcefully refresh the `task =
...
+> + The task operation can only be placed at the end of the dmctl command.
```
Comment (TomShawn) [inline reference/tools/data-migration/manage-tasks.md]:
```suggestion
> + The task operation can be placed only at the end of the dmctl command.
```

File: reference/tools/data-migration/manage-tasks.md (hunk 3 above)
Comment (toutdesuite) [reply reference/tools/data-migration/manage-tasks.md]: @TomShawn The author confirmed that the translation is ok. FYI

**Comment by TomShawn** [comment]
LGTM
@3pointer PTAL

File: reference/tools/data-migration/manage-tasks.md [hunk 6]
```diff
@@ -8,9 +8,11 @@ category: reference
 
 This document describes how to manage and maintain the data replication task using the [dmctl](/reference/tools/data-migration/overview.md#dmctl) component. For the Data Migration cluster deployed using DM-Ansible, the dmctl binary file is in `dm-ansible/dmctl`.
 
-## dmctl basic usage
+The dmctl component supports the interactive mode for manual operations, and also supports the command mode for the script.
 
-This section shows the basic usage of dmctl commands.
+## dmctl interactive mode
+
+This section described the basic use of dmctl commands in the interactive mode.
```
Comment (lilin90) [inline reference/tools/data-migration/manage-tasks.md]: @toutdesuite Why use the past tense?

File: reference/tools/data-migration/manage-tasks.md [hunk 7]
```diff
@@ -42,9 +44,15 @@ VjX8cEeTX+qcvZ3bPaO4h0C80pe/1aU=
 
 ### Task management overview
 
+Enters the interactive mode to interact with DM-master.
```
Comment (lilin90) [inline reference/tools/data-migration/manage-tasks.md]: `Enters` is weird. This is not a code comment.

File: reference/tools/data-migration/manage-tasks.md (hunk 6 above)
Comment (toutdesuite) [reply reference/tools/data-migration/manage-tasks.md]: @lilin90 The mistake is resolved in https://github.com/pingcap/docs/pull/2118. Thanks for pointing it out.

File: reference/tools/data-migration/manage-tasks.md (hunk 7 above)
Comment (toutdesuite) [reply reference/tools/data-migration/manage-tasks.md]: @lilin90 The mistake is resolved in #2118. Thanks for pointing it out.

**Pull request merged**